templates/trafegodns.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<Container version="2">
+    <Name>TrafegoDNS</Name>
+    <Repository>ghcr.io/elmerfds/trafegodns:latest</Repository>
+    <Registry>https://github.com/elmerfds/TrafegoDNS</Registry>
+    <Network>bridge</Network>
+    <Shell>bash</Shell>
+    <Privileged>false</Privileged>
+    <Support>https://github.com/elmerfds/TrafegoDNS/issues</Support>
+    <Project>https://github.com/elmerfds/TrafegoDNS</Project>
+    <Overview>TrafegoDNS automatically manages DNS records based on container configuration. It supports both Traefik integration and direct Docker container label mode, making it compatible with any web server or reverse proxy solution.
+    Key Features:
+    - Supports multiple DNS providers (Cloudflare, DigitalOcean, Route53)
+    - Two operation modes: Traefik API integration or direct Docker labels
+    - Optional cleanup of orphaned DNS records
+    - Real-time monitoring of Docker container events
+    - Please refer to the readme before deploying this container: https://github.com/elmerfds/TrafegoDNS</Overview>
+    <Category>Tools: Network:Management Status:Stable</Category>
+    <Icon>https://raw.githubusercontent.com/elmerfds/TrafegoDNS/main/logo/logo.png</Icon>
+    <DonateText>If you appreciate my work, please consider donating!</DonateText>
+    <DonateLink>https://github.com/elmerfds/TrafegoDNS</DonateLink>
+    <Config Name="Operation Mode" Target="OPERATION_MODE" Default="traefik|direct" Mode="" Description="Choose how to detect hostnames: Traefik API or direct container labels" Type="Variable" Display="always" Required="true" Mask="false">traefik</Config>
+    <Config Name="DNS Provider" Target="DNS_PROVIDER" Default="cloudflare|digitalocean|route53" Mode="" Description="Select DNS provider to use" Type="Variable" Display="always" Required="true" Mask="false">cloudflare</Config>
```

Comment:
```suggestion
    <Config Name="Operation Mode" Target="OPERATION_MODE" Default="traefik|direct" Mode="" Description="Choose how to detect hostnames: Traefik API or direct container labels" Type="Variable" Display="always" Required="true" Mask="false"/>
    <Config Name="DNS Provider" Target="DNS_PROVIDER" Default="cloudflare|digitalocean|route53" Mode="" Description="Select DNS provider to use" Type="Variable" Display="always" Required="true" Mask="false"/>
```

When using selectors, leave the field itself empty. The first item in the list will be the default.